onnx_chainer/functions/pooling.py

```diff
@@ -145,3 +149,43 @@ def convert_ROIPooling2D(func, opset_version, input_names,
         pooled_shape=[func.outh, func.outw],
         spatial_scale=func.spatial_scale,
     ),
+
+
+def convert_Unpooling2D(func, opset_version, input_names, num_outputs,
+                        parameters):
+    pad = [func.ph, func.pw]
+    stride = [func.sy, func.sx]
+    ksize = [func.kh, func.kw]
+    outsize = [func.outh, func.outw]
+    # TODO(hamaji): These could be implemented by `Slice` and `Pad`.
+    if func.cover_all:
+        raise RuntimeError('ONNX chainer does not support `cover_all=True` '
```

Comment:
Could you use "ONNX-Chainer" instead of "ONNX chainer" because current warning and error message use the phrase (I don't know reasons...), L172, L176, L180 are same.

Reply:
Done.